test/src/test/java/org/corfudb/integration/ServerRestartIT.java

```diff
@@ -733,6 +734,64 @@ public void testCorfuTableIndexReconstruction() throws Exception {
         // Stop the corfu server.
         assertThat(shutdownCorfuServer(corfuProcess)).isTrue();
     }
-}
 
+    @Test
+    public void testCorfuTableMultiIndexReconstruction() throws Exception {
```

Comment:
good test. Please add comments at the top. It helps build context faster when we come back to these things in future.

Reply:
Done.